acceptance/repository-postgresql/src/__tests__/postgresql-repository-execute.acceptance.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/test-repository-postgresql
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  DefaultTransactionalRepository,
+  Entity,
+  juggler,
+  model,
+  property,
+} from '@loopback/repository';
+import {expect, toJSON} from '@loopback/testlab';
+import {POSTGRESQL_CONFIG} from './postgresql.datasource';
+
+describe('PostgreSQL repository.execute()', () => {
+  // FIXME(bajtos) This test passes when executed in isolation, but fails
+  // on connection timeout when executed after repository-test suite.
+  it.skip('executes a parameterized native SQL query', async () => {
```

Comment:
Hmm interesting. It is indicative of some sort of bug somewhere (maybe a shared global state somehere?). I think we should resolve this.

Reply:
Sure, I agree this needs to be fixed. See https://github.com/strongloop/loopback-next/pull/6046#issuecomment-668457905

> I am proposing to skip the failing test for now and investigate the problem more either as part of #3342 or in a follow-up issue.

My primary motivation in this pull request is to ensure MongoDB commands can be executed. I don't want to keep this pull request open for too long, to avoid merge conflicts.

I am going to investigate the failing test as part of #3342, I added a sub-task to the TODO list.